### ISSUE TITLE
MS Word formatting is disturbed

### ISSUE THREAD

**Issue by ishanig11** [OfficeDev/office-js#4517]
I have updated ReactWordPortFolio sample to upload file. 
[Sample.zip](https://github.com/OfficeDev/office-js/files/15480174/Sample.zip)
[src.zip](https://github.com/OfficeDev/office-js/files/15480179/src.zip)
[SampleFile.docx](https://github.com/OfficeDev/office-js/files/15480181/SampleFile.docx)

Refer Sample.zip with application build. src.zip is source code. And SampleFile.docx is file we are trying to upload.
You can see loaded file in office online version and formatting is disturbed. Can you tell me how to handle this? This is show stopper issue.

**Comment by yilin4** [comment]
@ishanig11 Can you provide the project zip (the src's parent folder compression) to me? Since it seems that there is something lack when I use the src. Thank you~

**Comment by ishanig11** [comment]
[ReactWordPortFolio.zip](https://github.com/user-attachments/files/15508480/ReactWordPortFolio.zip)
execute npm install command to get npm modules installed.

**Comment by yilin4** [comment]
https://github.com/OfficeDev/office-js/assets/107384367/f5b32d37-0dd3-4f4d-bda8-dacc1558663a

It seems it works well on my work station. Can you provide a video or describe the behavior in details? Thanks.

**Comment by ishanig11** [comment]
@yilin4  : You can compare source document and document loaded using office.js. Here you can formatting is messed up. Second page information is coming on first page.

**Comment by yilin4** [comment]
@ishanig11 Sorry for responding late. I have tried and it was actually different from the sample file. I clicked the button at top right corner editing -> Open in desktop app to open the file at desktop and the file was same as the sample code, it seems that some of components in the sample file can't be rendered on online word. It has been put on our backlog<Bug#9053807> for internal track. We will keep track of this issue and let you know if there are any updates.
